cloud/shared/bin/lib/config_loader.py

```diff
@@ -37,6 +37,14 @@ def aws_region(self):
     def civiform_mode(self):
         return os.environ['CIVIFORM_MODE']
 
+    @property
+    def use_local_backend(self):
+        return os.environ['USE_LOCAL_BACKEND']
```

Comment:
I think this will throw error if the variable is not defined. Maybe use `os.environ.get('USE_LOCAL_BACKEND', False)`?

Reply:
done